code/ui/manager/src/components/sidebar/Search.tsx

```diff
@@ -28,6 +28,8 @@ const { document } = global;
 
 const DEFAULT_MAX_SEARCH_RESULTS = 50;
 
+export const FILTER_KEY = 'filter';
```

Comment:
I have a slight long-term concern here that this could conflict with the `experimental_setFilter` API, which we someday might want to expose in the URL too?

Could this be `'search'` instead maybe?

Reply:
renamed to `search` 👍